routers/repo/setting.go

```diff
@@ -367,6 +367,10 @@ func Collaboration(ctx *context.Context) {
 // CollaborationPost response for actions for a collaboration of a repository
 func CollaborationPost(ctx *context.Context) {
 	name := strings.ToLower(ctx.Query("collaborator"))
+	// name may be formatted as "username (fullname)"
+	if strings.Contains(name, "(") && strings.HasSuffix(name, ")") {
```

Comment:
nit: can simplify to
```
if index := strings.Index(name, " ("); index >= 0 {
	name = name[:index]
}

Reply:
I just cut-n-pasted @lunny's fix. I agree with you on this. Where can I put a utility method and then refactor all of the usages? Sorry new to Gitea, so want to make sure I'm following your guys' best practices... Thanks

Reply:
I think adding a `routers/utils/utils.go` file and putting it there makes the most sense. This proposed function seems specific to route handlers, so I'd prefer to not add it to `modules/util`.